ironblogger/static/style.css

```diff
@@ -81,6 +81,12 @@ section {
   overflow:hidden;
 }
 
+section > p > a {
```

Comment:
Like a rebase which flattens commits? I haven't done that before, actually,
but can try.

On Mon, Aug 24, 2015 at 12:52 PM, Ian Denhardt <notifications@github.com>
wrote:

> In ironblogger/static/style.css
> <https://github.com/zenhack/iron-blogger2/pull/46#discussion_r37773349>:
>
> > @@ -81,6 +81,12 @@ section {
> >    overflow:hidden;
> >  }
> >
> > +section > p > a {
>
> Or just rebase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zenhack/iron-blogger2/pull/46/files#r37773349>.
>